sdk/src/wallet/account/mod.rs

```diff
@@ -89,6 +90,8 @@ pub struct FilterOptions {
     pub foundry_ids: Option<HashSet<FoundryId>>,
     /// Return all nft outputs matching these IDs.
     pub nft_ids: Option<HashSet<NftId>>,
+    /// Return all outputs matching this address.
+    pub address: Option<Bech32Address>,
```

Comment:
Isn't that breaking change ?

Reply:
no, because it's additive

Reply:
I just tried, it's breaking change. Because all the field are public so if someone is building the whole thing with no default for remaining fields, it will break.

Reply:
This is not a CLI file

Reply:
oh whoops

Reply:
ah, right. Good catch then!